webpush/encrypt.go

```diff
@@ -139,36 +142,64 @@ type EncryptionResult struct {
 	ServerPublicKey []byte
 }
 
+// Decrypt an encrypted messages.
```

Comment:
What's the use case for adding decryption? AFAICT only the UA has any legitimate reason to decrypt the message.